### PR TITLE
docs(table): 6776 table example with overflow menu action

### LINKED ISSUE #6776: Table demo - add example using overflow-menu instead of dropdown + extra buttons
Mobile layout doesn't accommodate extra buttons in `pf-c-table__action`. The overflow menu was designed to handle this implementation.

Screen Shot 2022-01-13 at 9 58 06 AM

`.pf-c-table__action > .pf-c-table__overflow-menu`

### PR DESCRIPTION
**Pull request by gitdallas**
**What**: Closes #6776

### CODE REVIEW

**Comment by nicolethoen** [comment]
This is looking good!
Question for @mattnolting - it looks like at smaller widths, the width of the table is too wide for me to be able to see or scroll horizontally to the overflow menu. Should we update the width of the table examples at smaller views? If so, how?

**Review by kmcfaul — APPROVED** [review]

**Comment by mattnolting** [comment]
This looks like a workspace issue to me. Several intermediary content wrappers are causing odd issues, more detail coming soon...

**Comment by nicolethoen** [comment]
@gitdallas I will likely fix this the display issue at smaller widths on the core side.

**Review by nicolethoen — APPROVED** [review]

File: packages/react-table/src/components/TableComposable/examples/ComposableTableActionsOverflow.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <React.Fragment>
+      <TableComposable aria-label="Actions table">
+        <Thead>
+          <Tr>
+            <Th>{columnNames.name}</Th>
+            <Th>{columnNames.branches}</Th>
+            <Th>{columnNames.prs}</Th>
+            <Th>{columnNames.workspaces}</Th>
+            <Th>{columnNames.lastCommit}</Th>
+            <Td>{columnNames.overflowAction}</Td>
```
Comment (mattnolting) [inline packages/react-table/src/components/TableComposable/examples/ComposableTableActionsOverflow.tsx:68]: If the action table header cell has content, it is a `<th class="pf-c-table__action">`, and in this case I would add `pf-m-fit-content` here. 

Resulting in: `<th class="pf-c-table__action pf-m-fit-content">`

As an aside, `.pf-c-table__action` should be `text-align: right`. https://github.com/patternfly/patternfly/issues/4716

Comment (mattnolting) [reply packages/react-table/src/components/TableComposable/examples/ComposableTableActionsOverflow.tsx:68]: https://github.com/patternfly/patternfly/pull/4717

Comment (mattnolting) [reply packages/react-table/src/components/TableComposable/examples/ComposableTableActionsOverflow.tsx]: Disregard

**Review by mattnolting — APPROVED** [review]
LPTM! 🎉

File: packages/react-table/src/components/TableComposable/examples/ComposableTable.md [hunk 2]
```diff
@@ -167,6 +167,13 @@ To make a cell an action cell, render an `ActionsColumn` component inside a row'
 ```ts file="ComposableTableActions.tsx"
 ```
 
+### Composable: Actions Overflow
```
Comment (tlabaj) [inline packages/react-table/src/components/TableComposable/examples/ComposableTable.md:170]: Nit. This and should be sentence case.

**Pull request merged**